packages/core-mobile/app/hooks/earn/useIssueDelegation.ts

```diff
@@ -93,7 +106,9 @@ export const useIssueDelegation = (
       ) {
         return Promise.reject('invalid balance type.')
       }
-
+      if (pChainBalance.balancePerType.unlockedUnstaked === undefined) {
```

Comment:
@atn4z7, do we still need this?  I think I added this back when I rebased after you merge in your pr.

Reply:
no, we don't need it anymore but we shouldn't do the `pChainBalance.balancePerType.unlockedUnstaked || 0` below. we should let claimableBalance be undefined when unlockedUnstaked is undefined and adjust calculateAmountForCrossChainTransfer accordingly